docs/docs/guides/integrate/identity-providers/introduction.md

```diff
@@ -0,0 +1,182 @@
+
+
+- **Simplified account linking**: ZITADEL can link identities from multiple IdPs to a single user profile, facilitating a cohesive user experience across different authentication methods.
+
+
+- **Enhanced group and policy management**: With support for group synchronization and detailed policy configurations, ZITADEL offers granular control over user access and authentication processes.
+
+
+
+## The user journey 
+
+- **Integration with external identity providers**: ZITADEL supports integrating a variety of external identity providers, including social logins like Google or GitHub, as well as custom IdPs that use OpenID Connect or SAML protocols.
+
+
+- **Initiation of sign-in process**: Users select to sign in with an external IdP within your application.
+
+
+- **Redirection to ZITADEL**: The application redirects the user to ZITADEL, which guides them to their chosen external IdP for authentication.
+
+
+- **User authentication**: After successfully logging in at the external IdP site, users are redirected back to the application through ZITADEL, bringing along authentication tokens and profile information.
+
+
+- **Account linking for new users**: If the user's identity from the external IdP does not exist in ZITADEL, they're presented with two options before moving forward:
+  - **Create a new account**: Choosing this option creates a new ZITADEL account linked to the external IdP.
+  - **Autolinking with an existing local account**: Users have the option to link their new external identity to an existing local account in ZITADEL, enabling future logins with either their local account or the external IdP.
+
+
+- **Profile pre-filling from external IdP**: ZITADEL uses information from the external IdP to pre-fill the user's profile, simplifying the account creation or linking process.
+
+
+- **User option to update profile**: Users can review and, if necessary, update their pre-filled profile information.
+
+
+- **Session creation and access granting**: After the account is created or linked and the profile is set, the application grants access by creating a session for the user based on their authenticated identity.
+
+
+## Setting up external identity providers in ZITADEL
+
+
+In ZITADEL, you have the flexibility to link an external Identity Provider (IdP) to your entire instance, making it the default option for all organizations within your instance, or to connect it exclusively to a specific organization. This setup allows organization members to leverage similar capabilities in self-service if permitted.
+
+
+### Adjusting the custom login policy
+
+
+The login policy can be set as a default at the instance level and can be customized for each organization. The configuration process varies slightly depending on your focus:
+
+
+- **For instance-wide settings**, navigate to: `$YOUR-DOMAIN/ui/console/instance?id=general`
```

Comment:
I would name this default settings, as we are not really using instance settings anymore